cfg/cspell-dictionary.txt

```diff
@@ -3,7 +3,10 @@ auid
 bitshift
 collateralized
 econia
+lycheeverse
+micalevisk
```

Comment:
Since these are specific to the link checker, how about escaping them there? That way if we use a different link checker we won't have unused dict escapes?

Reply:
oh yes that makes much more sense, I should have realized that's what you were doing with the other ones, will update it